tests/publiccloud/registercloudguest.pm

```diff
@@ -75,4 +119,11 @@ sub post_fail_hook {
     $self->SUPER::post_fail_hook;
 }
 
+sub test_flags {
+    return {
+        fatal => 0,
+        milestone => 0
+    };
```

Comment:
```suggestion
    return { fatal => 0 };
```
I think the `milestone` is `0` by default. See [documentation](https://open.qa/docs/#_how_to_write_tests).